docs/source/aws.rst

```diff
@@ -0,0 +1,43 @@
+# How to ingest data from AWS
+
+Add AWS credentials to docker-compose
+
+Docker-compose has two repeated lines:
+
+AWS_ACCESS_KEY_ID=
+
+AWS_SECRET_ACCESS_KEY=
+
+You'll need to set those environment variables with your AWS access and secret key. And then you can call `docker-compose up`
+
+1)  Create a Provider
+
+Send a POST request to
+
+`http://<your_host>:8000/api/cost-management/v1/providers/`
+
+With the content:
+
+Provider Resource Name: <whatever your arn is>
+
+You should receive a 200 range response confirming that the Provider was created.
+
+2) Trigger Masu
+
+Send a GET request to
+`http://<your_host>:5000/api/cost-management/v1/download/`
+That _should_ trigger the worker to load the data into the DB. 
+
+You may want to check the logs to see if the loading is occuring:
+
+`docker-compose logs -f koku-server koku-worker`
+
+
+koku_beat is the Celery scheduler: Periodic Tasks — Celery 4.3.0 documentation
```

Comment:
Did you mean to add link here to the celery docs?

Reply:
no, it was more of a random piece of information that I wasn't sure where to put.